package.json

```diff
@@ -8,7 +8,7 @@
   },
   "homepage": "https://portal.gdc.cancer.gov/",
   "scripts": {
-    "preinstall": "npx npm-force-resolutions",
+    "preinstall": "npx npm-force-resolutions@0.0.3",
```

Comment:
Could you try one more thing?  Renamed `preinstall` to `force-resolve`.  This should remove `npm-force-resolutions` from the install stages, which should also solve this CI issue we're seeing.

My rationale is that builds should be deterministic and we should not force an update of dependencies at CI/CD build time.  Instead, this should be an intentional action done by a human, which can then be tested and reviewed.

Reply:
Yeah that does make sense. So are we okay with npm-force-resolutions not running at build time? I am guessing you can run it manually to force changes when needed?